sdk/identity/Azure.Identity/api/Azure.Identity.netstandard2.0.cs

```diff
@@ -198,6 +198,7 @@ public partial class InteractiveBrowserCredentialOptions : Azure.Identity.TokenC
         public Azure.Identity.AuthenticationRecord AuthenticationRecord { get { throw null; } set { } }
         public string ClientId { get { throw null; } set { } }
         public bool DisableAutomaticAuthentication { get { throw null; } set { } }
+        public string LoginHint { get { throw null; } set { } }
```

Comment:
Should this also be added to `DeviceCodeCredentialOptions`?

Reply:
I don't believe so